### PR TITLE
fix(metrics): Clean up metrics descriptions + rename 1 const

### PR DESCRIPTION
**Pull request by janpio**
When searching for information re https://prisma-company.slack.com/archives/C016KUHB1R6/p1693500424189099 I stumbled about some weird stuff in the definition of metrics descriptions:

- Some were duplicated
- They were grouped super weird
- Some were missing the initialization
- One constant used did not match the metric name

### CODE REVIEW

File: query-engine/metrics/src/lib.rs [hunk 1]
```diff
@@ -53,18 +53,21 @@ pub const MOBC_POOL_WAIT_COUNT: &str = "mobc_client_queries_wait";
...
 pub const PRISMA_POOL_CONNECTIONS_OPENED_TOTAL: &str = "prisma_pool_connections_opened_total";
 pub const PRISMA_POOL_CONNECTIONS_CLOSED_TOTAL: &str = "prisma_pool_connections_closed_total";
+// gauges
 pub const PRISMA_POOL_CONNECTIONS_OPEN: &str = "prisma_pool_connections_open";
 pub const PRISMA_POOL_CONNECTIONS_BUSY: &str = "prisma_pool_connections_busy";
 pub const PRISMA_POOL_CONNECTIONS_IDLE: &str = "prisma_pool_connections_idle";
 pub const PRISMA_CLIENT_QUERIES_WAIT: &str = "prisma_client_queries_wait";
+pub const PRISMA_CLIENT_QUERIES_ACTIVE: &str = "prisma_client_queries_active";
+// histograms
+pub const PRISMA_CLIENT_QUERIES_DURATION_HISTOGRAM_MS: &str = "prisma_client_queries_duration_histogram_ms";
```
Comment (janpio) [inline query-engine/metrics/src/lib.rs:68]: This was missing the `DURATION` before.

File: query-engine/metrics/src/lib.rs [hunk 2]
```diff
@@ -53,18 +53,21 @@ pub const MOBC_POOL_WAIT_COUNT: &str = "mobc_client_queries_wait";
 pub const MOBC_POOL_WAIT_DURATION: &str = "mobc_client_queries_wait_histogram_ms";
 
 // External metrics names that we expose.
+// counters
```
Comment (janpio) [inline query-engine/metrics/src/lib.rs:56]: grouping by type of metric will hopefully make this easier to grasp

File: query-engine/metrics/src/lib.rs [hunk 3]
```diff
@@ -118,6 +121,16 @@ pub fn setup() {
 // a new metric registry for a Query Instance the descriptions
 // will be in place
 pub fn describe_metrics() {
```
Comment (janpio) [inline query-engine/metrics/src/lib.rs:123]: This is completely resorted and now also grouped bxy metric type

File: query-engine/metrics/src/lib.rs [hunk 4]
```diff
@@ -145,31 +167,13 @@ pub fn describe_metrics() {
...
-
-    describe_gauge!(
-        PRISMA_POOL_CONNECTIONS_IDLE,
-        "Number of currently unused Pool Connections (waiting for the next datasource query to run)"
-    );
-
-    describe_gauge!(
-        PRISMA_CLIENT_QUERIES_WAIT,
-        "Number of Prisma Client queries currently waiting for a connection"
-    );
```
Comment (janpio) [inline query-engine/metrics/src/lib.rs:171]: These were duplicated before.

**Comment by Jolg42** [comment]
Note: 
For tests I think these are the related files
https://github.com/prisma/prisma-engines/blob/main/query-engine/connector-test-kit-rs/query-engine-tests/tests/new/metrics.rs
https://github.com/prisma/prisma-engines/blob/main/query-engine/black-box-tests/tests/metrics/smoke_tests.rs
https://github.com/prisma/prisma/blob/main/packages/client/tests/functional/metrics/enabled/tests.ts

**Review by tomhoule — APPROVED** [review]
Looks like a nice clean-up :100:

**Pull request merged**